includes/TinyMCE.php

```diff
@@ -104,9 +111,9 @@ public function enqueue_assets() {
 		$asset = $this->get_asset_metadata( 'tinymce-button' );
 
 		// Only works when wp_register_script and wp_enqueue_script are called separately.
-		wp_register_script( 'web-stories-tinymce-button', false, $asset['dependencies'], $asset['version'], true );
-		wp_enqueue_script( 'web-stories-tinymce-button' );
-		wp_add_inline_script( 'web-stories-tinymce-button', 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
+		wp_register_script( self::SCRIPT_HANDLE, false, $asset['dependencies'], $asset['version'], true );
+		wp_enqueue_script( self::SCRIPT_HANDLE );
+		wp_add_inline_script( self::SCRIPT_HANDLE, 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
```

Comment:
Could we actually use `wp_localize_script` now that we're doing `wp_register_script`?

Reply:
Changed.